### PR TITLE
PUB-1637 Delete Court

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link ###

https://tools.hmcts.net/jira/browse/PUB-1637

### Change description ###

Add a new page as part of System admin using which system admin can delete the court from reference data.

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: src/main/resources/locales/en/system-admin-dashboard.json [hunk 1]
```diff
@@ -12,6 +12,11 @@
       "description": "Upload CSV location reference data",
       "link": "manual-reference-data-upload"
     },
+    {
```
Comment (KianKwa) [inline src/main/resources/locales/en/system-admin-dashboard.json:15]: These should be added to the 'cy' folder too even though there is no Welsh. Same for the other resources

Comment (junaidiqbalmoj) [reply src/main/resources/locales/en/system-admin-dashboard.json:15]: added :) thanks.

**Review by KianKwa — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -67,12 +67,12 @@
...
+    "minimist": "^1.2.6",
     "moment": "2.29.4",
     "moment-timezone": "^0.5.37",
     "multer": "^1.4.5-lts.1",
     "node-blob": "^0.0.2",
-    "node-fetch": ">=3.2.10",
+    "node-fetch": ">=2.6.7",
     "nunjucks": "^3.2.2",
     "nunjucks-date-filter": "^0.1.1",
     "passport-azure-ad": "4.3.3",
```
Comment (joshblackmoor) [inline package.json:78]: Just wondering why two of these have been down graded compare to master?

Comment (junaidiqbalmoj) [reply package.json]: revert the change. frontend was broken on my machine, so trying to fix it and mistakenly push my packages.json file changes.

**Review by joshblackmoor — APPROVED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Review by joshblackmoor — APPROVED** [review]

**Review by ChrisS1512 — DISMISSED** [review]

**Review by joshblackmoor — APPROVED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Review by joshblackmoor — APPROVED** [review]

**Review by joshblackmoor — DISMISSED** [review]

**Review by joshblackmoor — APPROVED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**